integration-tests/cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-admin/DeviceManagement/FilterDeviceLogItem.feature

```diff
@@ -0,0 +1,96 @@
+@Common @Platform @AdminDeviceManagement
+Feature: Filter device messages
+  As an operator
+  I want to be filter and sort the device messages 
+  In order to analyse problems and check messages
+
+  Background: 
+    Given I have 6 device log items
+      | DeviceIdentification | DEV-1                |
+      | Organization         | Liander              |
+    And I have 5 device log items
+      | DeviceIdentification | DEV-2                |
+      | Organization         | Liander              |
+    And I have 4 device log items
+      | DeviceIdentification | DEV-11               |
+      | Organization         | TEST                 |
+
+  Scenario: No filters set so getting all the messages
+    When receiving a message log request without a filter
+    Then the messages response contains 15 messages with date filter or no filter
+
+  Scenario: Use wildcard filter for Device Identification
+    When receiving a filter message log request
+      | DeviceIdentification | DEV-1* |
+    Then the messages response contains 10 messages for devices
+      | DEV-1  |
+      | DEV-11 |
+
+  @NightlyBuildOnly
+  Scenario: Use wildcard filter for Organisation Identification
+    When receiving a filter message log request
+      | OrganizationIdentification | Li?nder |
+    Then the messages response contains 11 messages for devices
+      | DEV-1 |
+      | DEV-2 |
+
+  @NightlyBuildOnly
+  Scenario: Filter messages only on Organisation Identification
+    When receiving a filter message log request
+      | OrganizationIdentification | Liander |
+    Then the messages response contains 11 messages
```

Comment:
extend the step with 'for' to distinguish this step from the step where I requested to remove the filters

Reply:
Done.